pkg/transform/secrets/secrets.go

```diff
@@ -42,11 +42,32 @@ type MetaData struct {
 	Namespace string `yaml:"namespace"`
 }
 
+// SecretType is an enumerator for secret types
+type SecretType int
+
+const (
+	// Keystone - keystone type for Secret
+	Keystone = iota
```

Comment:
Usually the enum's typename is postfixed to these to prevent package level name collisions, since they're a little generic without it. So `KeystoneSecretType`, `HtpasswordSecretType` etc.

Reply:
Ok, thank you for advise, changed.